Tests/SwiftLintFrameworkTests/RulesTests.swift

```diff
@@ -9,7 +9,8 @@
 import SwiftLintFramework
 import XCTest
 
-// swiftlint:disable:next type_body_length
+// swiftlint:disable type_body_length
```

Comment:
Hmmm...I did a `brew upgrade swiftlint` and it updated to 0.13.2, but I'm still getting an error and a warning when I put this on a single line.